common/workunit/workunit.cpp

```diff
@@ -284,6 +296,13 @@ class CConstGraphProgress : public CInterface, implements IConstWUGraphProgress
                 tag = "node";
                 id += strlen(SubGraphScopePrefix);
                 break;
+            case SSTworkflow:
+                tag = "workflow";
+                id += strlen(WorkflowScopePrefix);
+                break;
+            case SSTgraph:
+                tag = "graph";
+                id += strlen(GraphScopePrefix);
```

Comment:
missing beak

Reply:
This section of code has been replaced. 'break' no longer appropriate.